cli/lib/kontena/cli/stack_command.rb

```diff
@@ -24,11 +20,8 @@ class Kontena::Cli::StackCommand < Kontena::Command
   subcommand "logs", "Show logs from services in a stack", Kontena::Cli::Stacks::LogsCommand
   subcommand "monitor", "Monitor services in a stack", Kontena::Cli::Stacks::MonitorCommand
   subcommand "build", "Build images listed in a stack file and push them to an image registry", Kontena::Cli::Stacks::BuildCommand
-  subcommand "push", "Push (or delete) a stack into the stacks registry", Kontena::Cli::Stacks::PushCommand
-  subcommand "pull", "Pull a stack from the stacks registry", Kontena::Cli::Stacks::PullCommand
-  subcommand "search", "Search for stacks in the stacks registry", Kontena::Cli::Stacks::SearchCommand
-  subcommand "info", "Show info about a stack in the stacks registry", Kontena::Cli::Stacks::InfoCommand
-  
+  subcommand "registry", "Stack registry related commands", Kontena::Cli::Stacks::RegistryCommand
```

Comment:
maybe add "reg" or something for shorthand? (or merge the thing that allows typing just enough to make subcommand  unique)